frontend/public/components/cloud-services/subscription.tsx

```diff
@@ -83,6 +83,7 @@ export const SubscriptionsPage: React.SFC<SubscriptionsPageProps> = (props) => <
 
 export const SubscriptionDetails: React.SFC<SubscriptionDetailsProps> = (props) => {
   const {obj, installedCSV, pkg} = props;
+  const catalogNS = _.get(obj.spec, 'sourceNamespace', olmNamespace);
```

Comment:
It's usually better to use

```js
const catalogNS = _.get(obj.spec, 'sourceNamespace') || olmNamespace;
```

Otherwise values like `null` or empty string won't get defaulted. `_.get` only defaults on `undefined`.

Not sure if that's possible in the API, but it doesn't hurt to handle those cases.

Reply:
Alright, fixed it.